client/blocks/signup-form/index.jsx

```diff
@@ -1387,8 +1330,6 @@ export default connect(
 			wccomFrom: getWccomFrom( state ),
 			isWoo: getIsWoo( state ),
 			isWooJPC,
-			isP2Flow:
-				isP2Flow( props.flowName ) || get( getCurrentQueryArguments( state ), 'from' ) === 'p2',
```

Comment:
In the codebase there are a bunch more checks where we check the if `from` query argument is `p2`. Let's clean those up too!

Here's one example:

https://github.com/Automattic/wp-calypso/blob/7236a424def0788ca8e64915c3b9c1097f802777/client/login/controller.js#L59

Reply:
Nice find! Cleaned by https://github.com/Automattic/wp-calypso/pull/101712/commits/fd2fdec3a50e90fb4f45620c7b3494b6c94d56c0, https://github.com/Automattic/wp-calypso/pull/101712/commits/994f242ecc36d5941097a9d398f6ddb8f38319de and https://github.com/Automattic/wp-calypso/pull/101712/commits/fba00de7b3b12b1725c918b6f2ea0091ec050399.